tests/integration/models/reddit/test_live.py

```diff
@@ -156,3 +156,14 @@ def test_close(self, _):
         with self.recorder.use_cassette(
                 'TestLiveThreadContribution_close'):
             thread.contrib.close()
+
+
+class TestLiveUpdateContribution(IntegrationTest):
+    @mock.patch('time.sleep', return_value=None)
```

Comment:
I thought sleep() may be called in between of `POST .../access_token` and `POST .../delete_update`. Since many other tests do not use mock.patch, I might be wrong.